coops-ui/src/resources/business-config.ts

```diff
@@ -0,0 +1,66 @@
+export class Flow {
+  feeCode: string
+  displayName: string
+  certifyText: string
+}
+
+export class Business {
+     typeEnum: string
+     displayName: string
+     flows: Array<Flow>
```

Comment:
Looks like there is an extra tab here.

Reply:
ah interesting...lint didn't pick this up, good catch!